v03_pipeline/lib/reference_data/compare_globals.py

```diff
@@ -34,9 +34,14 @@ def from_dataset_configs(
         rdc: ReferenceDatasetCollection,
         dataset_type: DatasetType,
         reference_genome: ReferenceGenome,
+        single_dataset: str | None = None,
     ):
         paths, versions, enums, selects = {}, {}, {}, {}
-        for dataset in rdc.datasets(dataset_type):
+        for dataset in (
```

Comment:
Each crdq uses 1 dataset. We don't need to build the Globals object with all datasets for the rdc (which will always be ReferenceDatasetCollection.COMBINED), so I added back this optional parameter

Reply:
would another option here be to pass in `datasets` as a `list[str]` rather than both the `rdc` and the `single_dataset`?

Reply:
yeah that works, I think that's what was done in the original implementation of this fn a few months back

Reply:
I changed the param to `datasets` as a `list[str]`